apps/web/src/modules/job/components/JobForm/hooks/useJobForm/index.ts

```diff
@@ -17,17 +22,30 @@ const useJobForm = (defaultValues: IPostJobSchemaType, edit?: boolean) => {
     reset(defaultValues)
   }, [defaultValues, reset])
 
+  const router = useRouter()
+  const { displaySnackbar } = useSnackbar()
+
   const { fields, append, remove } = useFieldArray({
     control,
     name: 'shooting',
   })
 
-  const handleSuccess: SubmitHandler<IPostJobSchemaType> = useCallback(() => {
-    console.log('success')
-    if (edit) {
-      console.log('edit success')
-    }
-  }, [edit])
+  const handleSuccess: SubmitHandler<IPostJobSchemaType> = useCallback(
+    async (data) => {
+      const payload = fieldToPayload(data)
+      console.log('success')
+      if (edit) {
+        const { jobId } = router.query as { jobId: string }
+        await apiClient.put<unknown, unknown, EditJobDto>(
+          `/job/${jobId}`,
+          payload,
+        )
```

Comment:
Do we need error handling ?

Reply:
will do that in the next sprint :cat_smile: